KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/KalturaEntryDataPersistence.java

```diff
@@ -98,19 +99,29 @@ public static Object getPropertyByEntry(String entryId, String subKey) throws Ex
 		return value;
 	}
 
+	public static Object setLock(String entryId) throws Exception {
+	    synchronized (entryIdToKalturaLiveEntryMap) {
+	        Object lock = new Object();
+            entryIdToKalturaLiveEntryMap.putIfAbsent(entryId, new ConcurrentHashMap<String, Object>());
+            ConcurrentHashMap<String, Object> entryMap = entryIdToKalturaLiveEntryMap.get(entryId);
+            entryMap.putIfAbsent(Constants.KALTURA_ENTRY_AUTHENTICATION_ERROR_FLAG, false);
```

Comment:
don't see the logic into putting this here... it doesn't belong to the logical nature of this method